examples/inference/README.md

```diff
@@ -0,0 +1,54 @@
+# Distributed inference examples with PiPPy
+
+This repo contains a variety of tutorials for using the [PiPPy](https://github.com/PyTorch/PiPPy) pipeline parallelism library with accelerate. You will find examples covering:
+
+1. How to trace the model using `accelerate.prepare_pippy`
+2. How to specify inputs based on what the model expects (when to use `kwargs`, `args`, and such)
+3. How to gather the results at the end.
+
+## Installation
+
+This requires the `main` branch of accelerate (or a version at least 0.27.0) and  `pippy` version of 0.2.0 or greater. Please install using `pip install .` to pull from the `setup.py` in this repo, or run manually:
+
+```bash
+pip install 'accelerate>=0.27.0' 'torchpippy>=0.2.0'
+```
+
+## Running code
+
+You can either use `torchrun` or the recommended way of `accelerate launch` on each script:
+
+```bash
+accelerate launch bert.py
```

Comment:
For accelerate, i think it could be great to let the user know that they can change the number of gpus that they are using by specifying --num_process= x with x the number of gpus. Also, we should probably specify that they don't need to use accelerate config since I personally think that it should only be used when training model for now.